### DISCUSSION TITLE
Can you also place the function as a picture?

### DISCUSSION THREAD

**Discussion by VanWolfing** [leonhma/obsidian-functionplot#30] [Ideas]
I don't know if this exists but it would be a great addition as a feature since currently it gives errors as I try to export the file.
Personally I just took a screenshot and replaced the codeblock.

**Comment by leonhma** [comment]
Yeah I've seen that functionality with Obsidian Charts (another plugin). I agree. Can you open a feature request for that? Issues > New Issue > Feature request

**Reply by VanWolfing** [reply]
I will do so
